src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+    static WordList wordList;
+
     public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        wordList = new WordList();
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        replyWelcomeMessage();
+        String input;
+        while (true) {
+            try {
+                input = sc.nextLine();
+                checkEmpty(input);
+                Object[] parseResult = InputParser.parseInput(input);
+                InputType inputType = (InputType) parseResult[0];
+                String[] value = (String[]) parseResult[1];
+
+                processInput(inputType, value);
+                if (inputType == InputType.BYE) {
+                    break;
+                }
+            } catch (EmptyInputException e) {
+                System.out.println(e);
+                continue;
+            } catch (EmptyDescriptionException e) {
+                System.out.println(e);
+            } catch (NumberFormatException e) {
+                System.out.println(e);
+            } catch (Exception e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+    public static void processInput(InputType inputType, String[] value) {
```

Comment:
Small thing because its just a parameter but maybe change value to values so its plural?